settings.gradle

```diff
@@ -76,24 +76,24 @@ dependencyResolutionManagement {
             def antlrVersion = version "antlr", "4.13.2"
             // WARNING: When upgrading to a version of bytebuddy that supports a new bytecode version,
             // make sure to remove the now unnecessary net.bytebuddy.experimental=true in relevant CI jobs (Jenkinsfile).
-            def byteBuddyVersion = version "byteBuddy", "1.17.6"
-            def checkerFrameworkVersion = version "checkerFramework", "3.49.5"
-            def classmateVersion = version "classmate", "1.7.0"
+            def byteBuddyVersion = version "byteBuddy", "1.17.7"
+            def checkerFrameworkVersion = version "checkerFramework", "3.51.0"
+            def classmateVersion = version "classmate", "1.7.1"
             def geolatteVersion = version "geolatte", "1.10"
             def hibernateModelsVersion = version "hibernateModels", "1.0.1"
-            def jandexVersion = version "jandex", "3.3.1"
-            def jacksonVersion = version "jackson", "2.19.1"
+            def jandexVersion = version "jandex", "3.5.0"
```

Comment:
requires a Jira ticket + needs to be updated in pair with hibernate-models to match the versions.